bokehjs/test/integration/regressions.ts

```diff
@@ -3744,4 +3744,26 @@ describe("Bug", () => {
       await display(p, [p.width! + left + 50, p.height! + top + 50])
     })
   })
+
+  describe("in issue #13692", () => {
+    describe("doesn't scale webgl antialising by pixel ratio", () => {
```

Comment:
```suggestion
    describe("doesn't scale webgl antialiasing by pixel ratio", () => {
```
Though this will require changes to baseline file names

Reply:
Done.